modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -19,7 +19,9 @@ GA:: General Availability
 
 | Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | |
```

Comment:
we are adding a new component "Manual Approval Gate" in 1.15, so we need to add here and version is 0.2.x (TP)